helm-charts/splunk-otel-collector/values.yaml

```
@@ -408,6 +408,9 @@ agent:
     # so that agent pods can be discovered via dns etc
     enabled: true
 
+    # (optional) Override the default service internal traffic policy
+    internalTrafficPolicy: "Local"
```

Can we make this more optional to not bloat the set values and be more portable? This would require updating the values.schema.json file.
```suggestion
    # internalTrafficPolicy: "Local"
```